src/tools/compiletest/src/header.rs

```diff
@@ -555,8 +550,8 @@ pub fn lldb_version_to_int(version_string: &str) -> isize {
     let error_string = format!("Encountered LLDB version string with unexpected format: {}",
                                version_string);
     let error_string = error_string;
-    let major: isize = version_string.parse().ok().expect(&error_string);
-    return major;
+    let major: isize = version_string.parse().expect(&error_string);
+    major
```

Comment:
Ah, I did not look at that code too closely. We had clippy ignore bindings with given type because we were getting some false positives around it. However, I agree, this should be one line. I'm not going to change APIs here...